floor_generator/pubspec.yaml

```diff
@@ -10,7 +10,7 @@ environment:
   sdk: '>=2.12.0 <3.0.0'
 
 dependencies:
-  analyzer: ">=2.1.0 <4.0.0"
+  analyzer: 4.1.0
```

Comment:
🙌 do you think it would make sense to make this version constraint a bit more flexible e.g. >= 4.0.0 <5.0.0?